starboard/doc/evergreen/cobalt_evergreen_overview.md

```diff
@@ -152,7 +160,36 @@ your platform unless it is not one of our supported architectures:
 If your target architecture falls outside the support list above, please reach
 out to us for guidance.
 
-#### Adding Crash Handlers to Evergreen
+#### Build commands
+
+As mentioned, the Google-maintained Evergreen toolchain is used to build Cobalt
+core (`libcobalt.so`). For example:
+
+```
+$ gn gen out/evergreen-arm-softfp_qa --args="target_platform=\"evergreen-arm-hardfp\" build_type=\"qa\" target_cpu=\"arm\" target_os=\"linux\" sb_api_version=15"
```

Comment:
target_platform should be evergreen-arm-softfp

Reply:
Fixed, thanks.